packages/caravan-fees/src/tests/rbf.fixtures.ts

```diff
@@ -3,179 +3,230 @@ import { UTXO, SCRIPT_TYPES } from "../types";
 
 export const rbfFixtures = {
   cancelRbf: [
-    // Mempool link: https://mempool.space/tx/a7d852d6997dcbbb3be82c5282d2951c6f75053dfef7d6444e0dffb6e7e6977c
+    // https://mempool.space/tx/be81b81620702718957d445611066bd596fe1840e219b84f6ea60e0114a7a305
     {
+      /*
+      ====================================== IMPORTANT ====================================
+      This multisig transaction example demonstrates a key aspect of RBF (Replace-By-Fee) cancellation:
+
+      Original Transaction:
+      - Type: P2WSH (Pay-to-Witness-Script-Hash)
+      - Inputs: 2
+      - Outputs: 3
+      - Fee: 20,880 satoshis
+      - Size: 348 vBytes
+      - Fee Rate: 60 sat/vB (20,880 / 348)
+
+      RBF Cancellation Scenario:
+      1. Target Fee Rate: 80 sat/vB (increased from original 60 sat/vB)
+      2. Cancellation tx typically has fewer outputs (usually just one to the cancellation address)
+      3. In this case, the new tx size decreases to 159 vBytes due to having only one output
+
+      RBF Fee Calculation:
+      - Minimum required fee: 20,880 satoshis (must be >= original fee to satisfy RBF rules)
```

Comment:
"to satisfy RBF rule #3"

Reply:
[done](https://github.com/caravan-bitcoin/caravan/pull/114/commits/09fba1f9947b8d3121995ab7d5bf94bf83423289)